SQLitePCLRaw.provider.sqlite3mc/generate.cmd

```diff
@@ -0,0 +1,4 @@
+powershell iwr https://github.com/ericsink/SQLitePCL.raw/raw/master/src/providers/provider.tt -OutFile provider.tt
+
+dotnet tool update -g dotnet-t4
+t4 provider.tt -o SQLite3Provider_sqlite3mc.cs -p NAME=sqlite3mc -p CONV=Cdecl -p KIND=dllimport -p NAME_FOR_DLLIMPORT=sqlite3mc -p FEATURE_WIN32DIR=FEATURE_WIN32DIR/false -p FEATURE_LOADEXTENSION=FEATURE_LOADEXTENSION/true -p FEATURE_FUNCPTRS=FEATURE_FUNCPTRS/false -p FEATURE_KEY=FEATURE_KEY/true
```

Comment:
Again to keep things simple, I'm using the same provider code everywhere (except for iOS which needs a different `DllImport` name.)